diff -up freeguide-0.10.10/src/freeguide/plugins/program/freeguide/options/OptionsDialog.java.orig freeguide-0.10.10/src/freeguide/plugins/program/freeguide/options/OptionsDialog.java
--- freeguide-0.10.10/src/freeguide/plugins/program/freeguide/options/OptionsDialog.java.orig	2008-01-18 22:32:34.000000000 +0100
+++ freeguide-0.10.10/src/freeguide/plugins/program/freeguide/options/OptionsDialog.java	2009-09-19 12:05:57.000000000 +0200
@@ -185,14 +185,6 @@ public class OptionsDialog extends FGDia
 
         trunk.add( browserLeaf );
 
-        panel = new PrivacyOptionPanel( this );
-
-        DefaultMutableTreeNode privateLeaf =
-            new DefaultMutableTreeNode( panel );
-        panel.construct(  );
-
-        trunk.add( privateLeaf );
-
         DefaultMutableTreeNode advancedBranch =
             new DefaultMutableTreeNode( 
                 Application.getInstance(  ).getLocalizedMessage( "advanced" ) );
diff -up freeguide-0.10.10/src/freeguide/plugins/program/freeguide/viewer/MainController.java.orig freeguide-0.10.10/src/freeguide/plugins/program/freeguide/viewer/MainController.java
--- freeguide-0.10.10/src/freeguide/plugins/program/freeguide/viewer/MainController.java.orig	2009-01-16 20:53:57.000000000 +0100
+++ freeguide-0.10.10/src/freeguide/plugins/program/freeguide/viewer/MainController.java	2009-09-19 12:06:11.000000000 +0200
@@ -236,12 +236,6 @@ public class MainController extends Base
 
         applicationFrame = mainFrame;
 
-        // Check the FreeGuide version
-        if( !"no".equals( FreeGuide.config.privacyInfo ) )
-        {
-            new VersionCheckerThread( getApplicationFrame(  ) ).start(  );
-        }
-
         //checkForNoData(  );
         mainFrame.waitForClose(  );
 
diff -up freeguide-0.10.10/src/freeguide/plugins/program/freeguide/wizard/FirstTimeWizard.java.orig freeguide-0.10.10/src/freeguide/plugins/program/freeguide/wizard/FirstTimeWizard.java
--- freeguide-0.10.10/src/freeguide/plugins/program/freeguide/wizard/FirstTimeWizard.java.orig	2009-01-14 03:56:11.000000000 +0100
+++ freeguide-0.10.10/src/freeguide/plugins/program/freeguide/wizard/FirstTimeWizard.java	2009-09-19 12:06:24.000000000 +0200
@@ -91,7 +91,6 @@ public class FirstTimeWizard
         panels.add( createRegionPanel(  ) );
         panels.add( createWorkingDirectoryPanel(  ) );
         panels.add( createBrowserPanel(  ) );
-        panels.add( createPrivacyPanel(  ) );
         panels.add( createInstallPanel(  ) );
 
         wizardFrame =
